src/main/resources/application.yml

```diff
@@ -49,6 +49,31 @@ jobs:
     name: "ProcessPrintResponses"
     cron: "0 15/30 * * * *" # every 30 minutes starting at 15 minutes past the hour - see analysis and recommendations in EIP1-2515
 
+temp-cert:
```

Comment:
Would it be worth fully spelling this? `temporary-certificate` ?

Reply:
I thought the two abbreviations of `temp` and `cert` were sufficiently well known not to cause confusion whilst providing a useful shortening.  But happy to remove any ambiguity and use full form of both 👍